cloud/IoTHubListener/EventHubListener.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.EventHubs;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using Microsoft.Azure.Devices;
+using Google.Protobuf;
+using HeartbeatProto;
+
+namespace GWManagementFunctions
+{
+    public static class EventHubListener
+    {
+        [FunctionName("EventHubListener")]
+        public static async Task Run(
+                [EventHubTrigger(

```

Comment:
Probably need some comments here about function source and destination ... making it super explicit for folks.

Reply:
One other thought here ... should we link to the MSDocs on setting up IoT Hub endpoints and the Routing docs in a comment?